bundle/manifests/opentelemetry.io_opentelemetrycollectors.yaml

```diff
@@ -49,39 +49,52 @@ spec:
           API.
         properties:
           apiVersion:
-            description: APIVersion defines the versioned schema of this representation
+            description: 'APIVersion defines the versioned schema of this representation
               of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values.
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
           kind:
-            description: Kind is a string value representing the REST resource this
+            description: 'Kind is a string value representing the REST resource this
               object represents. Servers may infer this from the endpoint the client
-              submits requests to. Cannot be updated. In CamelCase.
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           metadata:
             type: object
           spec:
             description: OpenTelemetryCollectorSpec defines the desired state of OpenTelemetryCollector.
             properties:
               additionalContainers:
-                description: AdditionalContainers allows injecting additional containers
-                  into the Collector's pod definition.
+                description: 'AdditionalContainers allows injecting additional containers
```

Comment:
Looks like 500 still isn't enough for some of these descriptions. Maybe we should shorten them instead of increasing the limit, though?